onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/AbstractGenericONPRC_EHRTest.java

```diff
@@ -103,6 +97,10 @@ protected void setEHRModuleProperties(ModulePropertyValue... extraProps)
         super._containerHelper.enableModule("ONPRC_Billing");
         super._containerHelper.enableModule("ONPRC_BillingPublic");
         super._containerHelper.enableModule("SLA");
+        goToEHRFolder();
+        super._containerHelper.enableModule("ONPRC_Billing");
+        super._containerHelper.enableModule("ONPRC_BillingPublic");
+        super._containerHelper.enableModule("SLA");
```

Comment:
We can save some navigations by enabling all of these modules at once.
```suggestion
        super._containerHelper.enableModules(List.of("ONPRC_Billing", "ONPRC_BillingPublic", "SLA"));
```